File: web3rpc/rpc-specs/code-samples/python/klay/block/getCouncilSize.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockTag = "0x1b4"

sdk = OpenSDK(host)
klay_response = sdk.klay.get_council_size(blockTag)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/block/getCouncilSize.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const blockNumberOrTag = '0x1b4'

    provider.klay.getCouncilSize({ blockNumberOrTag }, (err, data, response) => {
        console.log(data);
    });
}
)()